include/TTipHit.h

```diff
@@ -26,6 +27,8 @@ class TTipHit : public TGRSIDetectorHit {
     Double_t fSlowAmplitude;
     Double_t fGammaAmplitude;
    
+	bool csi_flag;
```

Comment:
Don't worry about it now, but consider making this a bitset like the other detector flags we have. The reason I say this is a bool takes a byte in a root file, so you may as well have the rest of that byte accessible for future things if you need more bools.